README.md

```diff
@@ -39,6 +39,26 @@ Install bluetooth and bring up local device:
         
 And you're ready to rock. You'll find the usual block documentation below.
 
+Method to connect to sensor tag
+--------------
+
+If during a fresh restart of the Pi (or even n.io - not 100% sure at this time)
+
+You must bring the Blue Tooth adapter down and back up for proper acquisition of BT LE sensortags, use the following commands:
+
+        $ sudo hciconfig hci0 down
+        $ sudo hciconfig hci0 up
+        $ sudo hciconfig hci0 leadv 3
+        $ sudo hciconfig hci0 piscan
+
+Then proceded with the following. (If just starting/restarting a service, skip to this step):
+
+1) start service
+2) after succesful start (and within 3 seconds of eachother)
+	a) press locate button on SensorTag
+	b) visit the following URL in browser: http://192.168.100.72:8181/services/BlueTooth_SensorTag/Test1/connect 
+		(URL is RasPi dependent, BlueTooh_SensorTag refers to service with SensorTagRead Block in it, Test1 refers to SensorTagRead Block)
+
```

Comment:
I will add in the explanation and update the URL format as mentioned.

I have tried the scan_and_connect version - and it appears the user will
still need to go through the whole

$ sudo hciconfig hci0 down
$ sudo hciconfig hci0 up
$ sudo hciconfig hci0 leadv 3
$ sudo hciconfig hci0 piscan

process to enable the bluetooth dongle - before a scan can take place. This
would be somewhat exclusive to a Raspberry pi of course, and may limit the
blocks future compatibility if done internally vs. through console input
independent of the service builder.

If we are considering using this device in the future - it would be nice to
somehow automate this connection process completely (make sure bluetooth is
enabled on device) to avoid having to send the connect command via url
input. I'm not 100% sure if this functionality was included originally -
but I don't remember Oren mentioning that it had this capability, but that
he would like it to.

With this 'autonomy', the only thing the user needs to enter into the block
is the SensorTag's MAC address which is printed on the device itself -
which is only important if the user decides to deploy multiple units
simultaneously.

-Steve

On Thu, Feb 12, 2015 at 5:22 PM, Chris Brackert <notifications@github.com>
wrote:

> In README.md
> <https://github.com/nio-blocks/sensor_tag/pull/3#discussion_r24634646>:
>
> > +
> > +You must bring the Blue Tooth adapter down and back up for proper acquisition of BT LE sensortags, use the following commands:
> > +
> > +        $ sudo hciconfig hci0 down
> > +        $ sudo hciconfig hci0 up
> > +        $ sudo hciconfig hci0 leadv 3
> > +        $ sudo hciconfig hci0 piscan
> > +
> > +Then proceded with the following. (If just starting/restarting a service, skip to this step):
> > +
> > +1) start service
> > +2) after succesful start (and within 3 seconds of eachother)
> > +	a) press locate button on SensorTag
> > +	b) visit the following URL in browser: http://192.168.100.72:8181/services/BlueTooth_SensorTag/Test1/connect
> > +		(URL is RasPi dependent, BlueTooh_SensorTag refers to service with SensorTagRead Block in it, Test1 refers to SensorTagRead Block)
> > +
>
> Using the connect command implies that the block (Test1) is configured to
> point to this specific Sensor Tag, right? It would be useful to explain
> this step as well. I believe it's something along the lies of running
> 'hciconfig' and inspecting the output for the Address.
>
> Without that step, wouldn't it make more sense to use scan_and_connect?
>
> Also, it may be helpful to reformat the url to something like:
> http://
> [[NIOHOST]]:[[NIOPORT]]/services/{{service_name}}/{{block_name}}/connect
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nio-blocks/sensor_tag/pull/3/files#r24634646>.
>